libODF_sbe_reader.py

```diff
@@ -1,68 +1,117 @@
-"""
-Code to break down a string of SBE raw data into parseable chunks to be converted by methods later on. No conversion to sci data is done here.
-"""
-
-#%pylab inline
-import xml.etree.ElementTree as ET
+#!/bin/bash/env
+'''
```

Comment:
Please follow docstring conventions outlined in the emails - sphinx, reStructured Text.